wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -71,20 +74,28 @@ export class SDKv2Route {
     const fromChainSupported = supportedChains.includes(fromContext.chain);
     const toChainSupported = supportedChains.includes(toContext.chain);
 
-    const supportedDestinationTokens = await this.rc.supportedDestinationTokens(
-      sourceToken,
-      fromContext.context,
-      toContext.context,
-    );
-
-    const toTokenSupported = !!supportedDestinationTokens.find((tokenId) => {
-      return isSameToken(tokenId, destToken);
-    });
+    if (!fromChainSupported || !toChainSupported) {
+      return false;
+    }
 
-    const isSupported =
-      fromChainSupported && toChainSupported && toTokenSupported;
+    try {
+      const cacheKey = `isRouteSupported-${sourceToken.address}-${fromChain}-${toChain}`;
+      const supportedDestinationTokens = await this.tokenCache.requestWithCache(
```

Comment:
I think we could DRY this up, by using this class's `supportedDestTokens` method here instead of hitting the cache directly.

Reply:
The only issue is that they pass in different parameters. I don't know exactly why that's the case, but I want to make sure it works both ways

Reply:
Fixed it up